cdk/domino_cdk/config/base.py

```diff
@@ -35,6 +35,18 @@ class DominoCDKConfig:
 
     install: dict
 
+    tags: Dict[str, str] = field_property(default={})
```

Comment:
Two things, both apparent if you run `python3 app.py generate_config_template`:

* The order the variables are defined in also reflects the order the have when the config is rendered, so I'd like to keep the tag declaration where it was (below `aws_account_id`) to maintain the existing order.
* `tags` now gets internally stored as `_tags`, so when the config is rendered into a yaml the field has the wrong name. :(

The unit tests also catch the last thing (it sees an extraneous config entry `_tags`, and doesn't error on lack of `tags`).

Reply:
Unfortunately you cannot have a class variable with default before variables without defaults. Perhaps I can assign defaults to the other variables to keep ordering.

Wait. Where your github CI disappeared? I'm already used to rely on unit-tests in PR

I'll see if we can guide yaml renderer to use tags. 

I really do not want to force all the consumers to be aware of one single field to be not usable and use a function instead.

Reply:
Err, I didn't see your comment and overlooked the defaults that were added. You can actually take the defaults off those variables, and remove the default that's on the tags variable. Since we're generating the templates from the config objects, we expect to have some value for everything anyway, and end enforcing schemas/supporting schema upgrades.

Reply:
Well, the defaults don't cause a problem and you might need it for the field property. The pop will still cause it to error if it's not there, so maybe disregard that last comment.

Reply:
yes. = field_property makes tags a wrapper function that calls getter and setter. It has to be there for all the property mechanics to work.

It is the same as @property decorator which is `name = property(getter, setter)` behind the scenes